app/indexers/descriptive_metadata_indexer.rb

```diff
@@ -20,7 +20,7 @@ def to_solr
       'sw_subject_temporal_ssim' => subject_temporal,
       'sw_subject_geographic_ssim' => subject_geographic,
       'sw_pub_date_facet_ssi' => pub_year,
-      'originInfo_date_created_tesim' => [creation_date].compact,
+      'originInfo_date_created_tesim' => creation_date,
```

Comment:
Arcadia told me she preferred a single value.  I know the field can take multiple values.